modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -284,6 +284,12 @@ definition {
 		PortletEntry.save();
 	}
 
+	macro addObjectFolderViaAPI {
+		JSONObjectFolder.addFolder(
+			folderLabel = ${folderLabel},
+			folderName = ${folderName});
+	}
+
```

Comment:
Hey @JhossephG, we can remove this and the other commits from Luiz because we discussed that we should just call the macro directly instead of wrapping it in another macro